src/GrpcProtos/capture.proto

```diff
@@ -64,6 +64,7 @@ message CaptureOptions {
   DynamicInstrumentationMethod dynamic_instrumentation_method = 18;
 
   repeated InstrumentedFunction instrumented_functions = 5;
+  map<uint64, uint64> absolute_address_to_size_of_functions_to_stop_at = 19;
```

Comment:
As hinted in the other comment, here we don't really know what "stop" means. I think we should add "unwinding": `functions_to_stop_unwinding_at`.

Reply:
Agree. Actually had this in my original draft, but then copy&pasted the new names to here. Done.